Arrow icons customization

sesquialtera87 opened the pull request:
It would be nice to have a way to change the icons appearing in the description window.
In may case, I'm using FlatLaf and I prefer to change the default icons of the `AutoCompleteDescWindow` with other flat icons.
Currently I didn't find any solution, since also `backAction` and `forwardAction` are private fields, and so subclass `AutoCompleteDescWindow` it's not helpful.

I propose to search for the icons into the `UIManager`, so that one can customize them simply through `UIManager.put("autocomplete.leftArrow", icon)` and `UIManager.put("autocomplete.rightArrow", icon)`.

During initialization, `ToolBarForwardAction` and `ToolBarBakcAction` search for custom icons and, if not found, use the default currently provided by the library.

sesquialtera87 committed:
Search for left/right icons into UIManager to allow customization

sesquialtera87 committed:
Search for left/right icons into UIManager to allow customization

sesquialtera87 committed:
Some fix

sesquialtera87 committed:
Fix wrong UI property name

sesquialtera87 committed:
Fix UI property value

sesquialtera87 committed:
fix: checkstyle issues

bobbylight commented:
@sesquialtera87 can you rebase and fix the checkstyle issues (I think the workflows should run now)? 🙏

sesquialtera87 committed:
refactor: changes in UIManager keys and minimal docs

sesquialtera87 commented:
I’ve rebased the branch onto the latest `master `and fixed all the Checkstyle violations.
The build now completes successfully without any style issues. Let me know if anything else needs to be adjusted.

bobbylight commented:
Thanks for the PR!
